### ISSUE TITLE
Not compatible with newest Penumbra release.

### ISSUE THREAD

**Issue by ghost** [KazWolfe/XIVDeck#24]
Hello!

I just logged in  and got this message after updating Penumbra.

[<a href="https://imgur.com/ZogC9Yi"><img src="https://i.imgur.com/ZogC9Yi.png" title="source: imgur.com" /></a>](url)

**Comment by KazWolfe** [comment]
Hi! Thank you for the report. I published XIVDeck [version 0.2.6](https://github.com/KazWolfe/XIVDeck/releases/tag/v0.2.6) earlier today which should fix this bug.

As this update includes some other changes, I've decided to push this to [Dalamud Testing](https://goatcorp.github.io/faq/dalamud_troubleshooting.html#q-how-do-i-enable-plugin-test-builds) to put it through a test release. Assuming nobody reports any issues with it in the next few days, I'll push this release out to everyone. If you'd like to get this fix early, I'd recommend enabling testing mode. If not, just hang tight for a few days.

Thanks!
